.github/CODEOWNERS

```diff
@@ -280,6 +279,7 @@
 /comp/trace/etwtracer @DataDog/windows-agent
 /comp/autoscaling/datadogclient @DataDog/container-integrations
 /comp/etw @DataDog/windows-agent
+/comp/languagedetection/client @DataDog/container-platform
```

Comment:
This change is needed by the linter. It's introduced by `inv components.lint-components --fix`.